<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RichTextEditWidget</class>
 <widget class="QWidget" name="RichTextEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QFontComboBox" name="fontCombo">
       <property name="toolTip">
        <string>Font</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sizeCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Font Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="leftAlignButton">
       <property name="toolTip">
        <string>Left Alignment</string>
       </property>
       <property name="text">
        <string>Left</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Left.png</normaloff>:/NoteBook/Resources/Left.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="centerAlignButton">
       <property name="toolTip">
        <string>Center Alignment</string>
       </property>
       <property name="text">
        <string>Center</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Center.png</normaloff>:/NoteBook/Resources/Center.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="rightAlignButton">
       <property name="toolTip">
        <string>Right Alignment</string>
       </property>
       <property name="text">
        <string>Right</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Right.png</normaloff>:/NoteBook/Resources/Right.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="boldButton">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Bold (Ctrl+B)</string>
       </property>
       <property name="text">
        <string>B</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Bold.png</normaloff>:/NoteBook/Resources/Bold.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+B</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="italicButton">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="toolTip">
        <string>Italic (Ctrl+I)</string>
       </property>
       <property name="text">
        <string>I</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Italic.png</normaloff>:/NoteBook/Resources/Italic.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+I</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="underlineButton">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="toolTip">
        <string>Underline (Ctrl+U)</string>
       </property>
       <property name="text">
        <string>u</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Underline.png</normaloff>:/NoteBook/Resources/Underline.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+U</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="strikethroughButton">
       <property name="toolTip">
        <string>Strikethrough</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Strikethrough.png</normaloff>:/NoteBook/Resources/Strikethrough.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bulletTableInsertButton">
       <property name="toolTip">
        <string>Bulleted List</string>
       </property>
       <property name="text">
        <string>Table</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Bullet Table.png</normaloff>:/NoteBook/Resources/Bullet Table.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="numberTableInsertButton">
       <property name="toolTip">
        <string>Numbered List</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Number Table.png</normaloff>:/NoteBook/Resources/Number Table.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tableButton">
       <property name="toolTip">
        <string>Table</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Table.png</normaloff>:/NoteBook/Resources/Table.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="insertHLineButton">
       <property name="toolTip">
        <string>Horizontal Line</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Horizontal Line.png</normaloff>:/NoteBook/Resources/Horizontal Line.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="styleButton">
       <property name="toolTip">
        <string>Style</string>
       </property>
       <property name="text">
        <string>Apply Style...</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="CColorButton" name="textColorButton">
       <property name="toolTip">
        <string>Text (Foreground) Color</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Text Foreground.png</normaloff>:/NoteBook/Resources/Text Foreground.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="CColorButton" name="textBackgroundButton">
       <property name="toolTip">
        <string>Background Color</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Text Background.png</normaloff>:/NoteBook/Resources/Text Background.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="clearFormattingButton">
       <property name="toolTip">
        <string>Clear all formatting</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/clear-formatting.png</normaloff>:/NoteBook/Resources/clear-formatting.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="indentLeftButton">
       <property name="toolTip">
        <string>Reduce Indent</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/IndentLeft.png</normaloff>:/NoteBook/Resources/IndentLeft.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="indentRightButton">
       <property name="toolTip">
        <string>Increase Indent</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/IndentRight.png</normaloff>:/NoteBook/Resources/IndentRight.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="searchButton">
       <property name="toolTip">
        <string>Search (Alt+S)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="pynotebook.qrc">
         <normaloff>:/NoteBook/Resources/Search.png</normaloff>:/NoteBook/Resources/Search.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CustomTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QWidget" name="richTextEditSearchWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QPushButton" name="searchHideButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/PyNoteBook/Resources/Close.png</normaloff>:/PyNoteBook/Resources/Close.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="prevButton">
        <property name="toolTip">
         <string>Previous Occurrence (Ctrl+P)</string>
        </property>
        <property name="text">
         <string>Previous</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+P</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="toolTip">
         <string>Next Occurrence (Ctrl+N)</string>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+N</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CColorButton</class>
   <extends>QToolButton</extends>
   <header>ColorButton.h</header>
  </customwidget>
  <customwidget>
   <class>CustomTextEdit</class>
   <extends>QTextEdit</extends>
   <header>custom_text_edit</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pynotebook.qrc"/>
 </resources>
 <connections/>
</ui>
